core-bundle/src/HttpKernel/ModelArgumentResolver.php

```diff
@@ -84,7 +84,6 @@ private function fetchModel(Request $request, ArgumentMetadata $argument): ?Mode
             return $GLOBALS['objPage'];
         }
 
-        /** @var Model $model */
```

Comment:
The type hint is required for PHPStorm:

<img width="448" alt="" src="https://user-images.githubusercontent.com/1192057/143822096-b02ad1ce-febd-43fe-89c0-766b6dc06f11.png">

Reply:
Even if we revert https://github.com/leofeyer/contao/pull/3#discussion_r758078846 ?

Reply:
Yes, unfortunately. This is what PHPStorm thinks `$model` is without the annotation:

<img width="560" alt="Bildschirmfoto 2021-11-29 um 10 16 57" src="https://user-images.githubusercontent.com/1192057/143840388-edac65cc-430e-404f-b5c2-0c33a4cc6060.png">

Reply:
Fixed in bc2e794db1918a674f997688af1d8dfc81f4783f